payjoin/src/send/mod.rs

```diff
@@ -30,12 +30,19 @@ pub mod v2;
 
 type InternalResult<T> = Result<T, InternalProposalError>;
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[cfg_attr(feature = "v2", derive(serde::Serialize, serde::Deserialize))]
+pub struct AdditionalFeeContribution {
```

Comment:
Is there any good reason for this to be `pub`?

Reply:
Nope. Thanks for catching that. Switched to pub(crate)